### ISSUE TITLE
Element doesn't finish startup tasks until the tab is in focus

### ISSUE THREAD

**Issue by Half-Shot** [vector-im/element-web#23128]
### Steps to reproduce

1. (Presumably) have a "large" account with thousands of rooms.
2. Start element in a tab as normal.
3. Note that it starts out with a "blank" interface, the structure is there but the timeline/room list is empty.
4. Switch away to another tab for 5+ minutes
5. Notice when switching back, the interface is still blank
6. Leave the tab focused, eventually Element loads.

#### What did you expect?

Element should be able to complete it's startup processes and be "ready" without needing the tab to be focused. It's annoying to have to make the user sit and wait for a task when they could be doing other things.

#### What happened instead?

Element would only properly start when the tab was focused, meaning I couldn't use anything else in my browser.

### Operating system

Arch Linux / Wayland w/ Sway

### Browser information

Firefox 103

### URL for webapp

develop.element.io

### Application version

Element version: a302dba006f5-react-ef0ba77f1446-js-3ae974e23e41 Olm version: 3.2.12

### Homeserver

Synapse / half-shot.uk

### Will you send logs?

Yes

**Comment by t3chguy** [comment]
This will most likely be a browser "feature" and not one Element can work around
Element doesn't handle itself differently when out of focus by intent

**Comment by Half-Shot** [comment]
Indeed, this might be more useful as a tracking issue for people to see than something we can fix.
